superbench/benchmarks/micro_benchmarks/cuda_nccl_bw_performance.py

```diff
@@ -88,6 +88,12 @@ def add_parser_arguments(self):
             default=5,
             help='Number of warmup iterations. Default: 5.',
         )
+        self._parser.add_argument(
+            '--graph_iters',
+            type=int,
+            default=0,
+            help='Number of graph launch iterations. Set to 0 to disable graph mode. Default: 0.',
```

Comment:
Also it's not related to latency as well

Reply:
Fixed in PR title